### DISCUSSION TITLE
apt-get install error

### DISCUSSION THREAD

**Discussion by massimogallina** [kairos-io/kairos#1863] [Q&A] [answered]
I would like to use and customize `quay.io/kairos/kairos-debian:v2.4.0-k3sv1.27.3-k3s1`; but if I try to install new packages, I receive the error: `E: Archives directory /var/cache/apt/archives/partial is missing. - Acquire (2: No such file or directory)`.

This is the process to reproduce the issue:

```sh
docker run -it --entrypoint sh --rm quay.io/kairos/kairos-debian:v2.4.0-k3sv1.27.3-k3s1
# sudo apt-get install vim
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libsodium23 vim-common vim-runtime xxd
Suggested packages:
  ctags vim-doc vim-scripts
E: Archives directory /var/cache/apt/archives/partial is missing. - Acquire (2: No such file or directory)
```

If I do a `mkdir -p /var/cache/apt/archives/partial` and retry to install the package, it works.

**Comment by mauromorales** [comment, marked as answer]
Hi @massimogallina, thanks for reaching out. Indeed we started encountering this bug with Kairos v2.4.0, not sure why, since apt used to work despite `/var/cache` being removed (and it should work since it's a directory that is expected to be removed at anytime). We have a fix already on master and the patch version should land some time this or next week.

relates to: https://github.com/kairos-io/kairos/issues/1849
